### PR TITLE
Add link to slack invitations to README

### PR DESCRIPTION
**Pull request by jhawthorn**

### CODE REVIEW

**Comment by jordan-brough** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 ![](https://raw.githubusercontent.com/solidusio/solidus/master/solidus.png)
 
 * [solidus.io](http://solidus.io/)
+* [Join our slack](http://slack.solidus.io/) ([solidusio.slack.com](http://solidusio.slack.com))
```
Comment (magnusvk) [inline README.md]: <nitpick>I would capitalize Slack here</nitpick> (also, ignore me)

**Comment by magnusvk** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (jhawthorn) [inline README.md]: done

**Comment by magnusvk** [comment]
:smile: thank you

**Pull request merged**